milli/src/search/query_tree.rs

```diff
@@ -498,7 +510,9 @@ fn create_query_tree(
                 .filter(|(_, part)| !part.is_phrase())
                 .max_by_key(|(_, part)| match part {
                     PrimitiveQueryPart::Word(s, _) => {
-                        ctx.word_documents_count(s).unwrap_or_default().unwrap_or(u64::max_value())
+                        let (pair_freq, _, _) =
+                            split_best_frequency(ctx, s).unwrap_or_default().unwrap_or_default();
+                        pair_freq
```

Comment:
Oh! Sorry, I forgot this part of the code 🙏.
Could you come back to the previous version, please?

Reply:
Sure, I will add back word_documents_count 👍 and revert this part

Reply:
Done, PTAL.